io.github.prateekmedia.flutube.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.prateekmedia.flutube
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: flutube
+rename-appdata-file: flutube.appdata.xml
+rename-desktop-file: flutube.desktop
+command: flutube
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=~/FluTube:create
```

Comment:
Any reason it doesn't use `xdg-download` for storing files?

Reply:
No concrete reason, just to keep youtube downloaded music and videos seperate from normal downloads.

Reply:
I think polluting home directory is generally considered bad practice. Common downloads directory exists exactly for that use case.

Reply:
Okay, I will then create a sub-directory inside `Downloads` named `FluTube` to keep them seperate.